loopchain/channel/channel_inner_service.py

```diff
@@ -25,7 +13,8 @@
 
 from loopchain import configure as conf
 from loopchain import utils as util
-from loopchain.baseservice import BroadcastCommand, BroadcastScheduler, BroadcastSchedulerFactory, ScoreResponse
+from loopchain.baseservice import BroadcastCommand, BroadcastScheduler, BroadcastSchedulerFactory, \
+    ScoreResponse
```

Comment:
Can you explain why did you break the line?

Reply:
You got it.